lib/IDE/CodeCompletion.cpp

```diff
@@ -2778,7 +2786,8 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
     if (ParentKind != StmtKind::If && ParentKind != StmtKind::Guard)
       return;
     CodeCompletionResultBuilder Builder(Sink, CodeCompletionResult::ResultKind::Keyword,
-      SemanticContextKind::ExpressionSpecific, expectedTypeContext);
+      SemanticContextKind::Local, expectedTypeContext);
```

Comment:
Why `Local`?  I would have expected `None`.

Reply:
Not strong reason for `Local` or `CurrentModule`. But I couldn't just use `None` because `None` is rather deprioritized in `CodeCompletionOrganizer`. Maybe we need a new "neutral" semantic context kind.

Reply:
I'm fine with leaving it the way you wrote for the current PR, but I think we should leave a comment about it and we should probably figure out a better solution.  Either change the scoring to be more specific so that `None` is okay for these cases, or else introduce new context(s).